d03a/finance/AUTHOR.json

```diff
@@ -1 +1 @@
"UNS", "tag":"SECTION_CONTROL", "truncatable":true, "minOccurances":0, "maxOccurances":1, "fields":[{"tag":"code", "repeat":false, "required":true, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}, {"tag":"Section_identification", "repeat":false, "required":true, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}]}, "VLI":{"code":"VLI", "tag":"VALUE_LIST_IDENTIFICATION", "truncatable":true, "minOccurances":0, "maxOccurances":1, "fields":[{"tag":"code", "repeat":false, "required":true, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}, {"tag":"VALUE_LIST_IDENTIFICATION", "repeat":false, "required":true, "truncatable":true, "dataType":"composite", "startIndex":-1, "length":-1, "components":[{"tag":"Value_list_identifier", "required":true, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Object_identification_code_qualifier", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}]}, {"tag":"PARTY_IDENTIFICATION_DETAILS", "repeat":false, "required":false, "truncatable":true, "dataType":"composite", "startIndex":-1, "length":-1, "components":[{"tag":"Party_identifier", "required":true, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Code_list_identification_code", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Code_list_responsible_agency_code", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}]}, {"tag":"Status_description_code", "repeat":false, "required":false, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}, {"tag":"Value_list_name", "repeat":false, "required":false, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}, {"tag":"Designated_class_code", "repeat":false, "required":false, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}, {"tag":"Value_list_type_code", "repeat":false, "required":false, "truncatable":true, "dataType":"string", "startIndex":-1, "length":-1, "components":[]}, {"tag":"CHARACTERISTIC_DESCRIPTION", "repeat":false, "required":false, "truncatable":true, "dataType":"composite", "startIndex":-1, "length":-1, "components":[{"tag":"Characteristic_description_code", "required":true, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Code_list_identification_code", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Code_list_responsible_agency_code", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Characteristic_description", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}, {"tag":"Characteristic_description_1", "required":false, "truncatable":true, "dataType":"string", "subcomponents":[]}]}, {"tag":"Maintenance_operation_code", "repeat":false, "requ
```

Comment:
Let's hard code edifact header and footer segments as ignore segments in all schemas, untill we add support for those/

Reply:
done